README.md

```diff
@@ -626,14 +626,14 @@ branchUniversalObject.generateShortUrl(this, linkProperties, new BranchLinkCreat
 });
 ```
 ### List links in Google Search with App Indexing
-Getting your Branch link and app content listed in Google search is very easy with BranchUniversalObject. Once you've created the BUO, use the following API to list your app contents in Google Search via their App Indexing API. Your app will be opened with deep link data upon user clicking the search result and the session will be tracked.
+Getting your Branch link and app content listed in Google search is very easy with BranchUniversalObject. Once you've created the BUO, use the following API to list your app contents in Google Search via Firebase App Indexing API. Your app will be opened with deep link data upon user clicking the search result and the session will be tracked.
 
 ```java
 branchUniversalObject.listOnGoogleSearch(context);
 ```
-Note : Please make sure you have added the following to your gradle dependency
+Note : Please make sure Firebase app indexing is enabled for your application and has Firebase app indexing dependency added
 ```java
-'com.google.android.gms:play-services-appindexing:9.+'
+'compile 'com.google.firebase:firebase-appindexing:10.0.1'
 ```
```

Comment:
Also, we should make sure we update our official docs https://github.com/BranchMetrics/documentation/blob/master/pages/marketing-channels/google-app-indexing.md